server_extensions_extra/src/test/java/com/generalbytes/batm/server/extensions/BatmExtensionsXmlTest.java

```diff
@@ -59,7 +59,7 @@ public void testExtensionClassesExist() throws Exception {
     @Test
     public void testCryptoCurrencyValidators() throws Exception {
         // currencies mentioned in batm_extensions.xml but having CryptoCurrencyValidators implemented in the non-opensourced part of the codebase
-        Set<String> supportedInBuiltin = ImmutableSet.of("TRTL", "BTC", "LBTC", "XMR", "ETC");
+        Set<String> supportedInBuiltin = ImmutableSet.of("TRTL", "BTC", "LBTC", "XMR", "ETC","EGLD","USDTTRON");
```

Comment:
EGLD address validator is being added in this PR so the test should pass for it without adding the exception here.
USDTTRON address validator (TronCryptoAddressValidator) was already present in server_extensions_extra.
Could you try to run the test without adding "EGLD","USDTTRON" ? Are you getting an error message?

Reply:
Done . All tests passed!